docs/changelog.txt

```diff
@@ -35,6 +35,7 @@ Template for new versions:
 ## Documentation
 
 ## API
+- ``Units::setAutomaticProfessions``: bay12-provided entry point to assign labors based on work details
```

Comment:
plz move down to Future section -- this is the template : )

Reply:
oops :)